cli/shared/src/main/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/Deployment.scala

```diff
@@ -46,7 +47,8 @@ object Deployment {
     noOfReplicas: Int,
     externalServices: Map[String, Seq[String]],
     deploymentType: DeploymentType,
-    jqExpression: Option[String],
+    jsonTransform: JsonTransform,
+    jqExpression: Option[JsonTransformExpression],
```

Comment:
I think is a right direction to take, but I'm worried about verbosity of keeping both jsonTransform and jqExpression everywhere, also passing both as arguments to functions.

What might be the simplest thing is:
`jsonTransform: Json => Future[Json]`

And then jq can make this function for you?
`jq.makeTransform(JsonTransformExpression(".test = 123"))`
or even
`jq.makeTransform(".test = 123")`

Reply:
Yeah I think unifying these to into one parameter is the right way to go.

Particularly given when I went and specified `jq` as the `JsonTransform` so it can handle a `None` transformation.. 😄

I'll fix this possibly tomorrow.